library/std/src/io/error.rs

```diff
@@ -1,14 +1,14 @@
 #[cfg(test)]
 mod tests;
 
-#[cfg(target_pointer_width = "64")]
+#[cfg(all(target_pointer_width = "64", not(target_os = "uefi")))]
```

Comment:
Why was this necessary? Mind adding a comment in a new PR?

Reply:
RawOsError is 64-bit on UEFI. So we cannot use packed representation which assumes 32-bit os error: https://github.com/rust-lang/rust/pull/105861#pullrequestreview-1560771094

I will create PR adding the comment.